test/snippet/range/container/small_string.cpp

```diff
@@ -0,0 +1,18 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/container/small_string.hpp>
+
+int main()
+{
+    constexpr seqan3::small_string sm{"hello"};     // construct from string literal at compile time!
+
+    static_assert(sm[0] == 'h');                    // This way I can also test it at compile-time
+
+    seqan3::debug_stream << sm.size() << std::endl; // prints 5! (the null character is only stored internally)
+
+    // conversion to a normal string:
+    std::string sm_string{sm.string()};
+    // conversion to a cstring:
+    const char * sm_cstr{sm.c_str()};
```

Comment:
This is not a conversion strictly speaking...

Reply:
Also **please take care of east const**

Reply:
changed it to `// access data directly with a pointer to the underlying zero-terminated array:`